### PR TITLE
Update to use Sawtooth 1.0 SDK

### PR DESCRIPTION
**Pull request by grkvlt**
- Fix docker files to use latest images and commands
- Updates processor to use latest messages
- Change ledger sync to use new client state events

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -39,8 +39,8 @@ services:
 
   rethink:
     image: rethinkdb:2.3
-    expose:
-      - 28015
+    ports:
+      - "9090:8080"
```
Comment (boydjohnson) [inline docker-compose.yaml]: Hi @grkvlt Thanks for your work on this. Does the RethinkDb container need to link ports with the host and are these the correct ports?

Comment (grkvlt) [reply docker-compose.yaml]: This is for the RethinkDB dashboard, so I exposed it to have a look at how it works. I think it's fine for the compose files, obviously you'd do something a bit more secure for a production setup...

**Comment by boydjohnson** [comment]
@grkvlt When running both `docker-compose.yaml` and `docker-compose-dev.yaml` as the backend and then in the ui directory `npm start -- env=prod` I can signup with a user but do nothing else. That user gets processed on the blockchain and I see the state delta event put the user in the RethinkDb database. It seems like a UI configuration problem to me. When I click on the `create group` button on the create group modal, I don't see any network activity. I just wanted to let you know that I ran it locally.

**Comment by grkvlt** [comment]
@boydjohnson I see that as well, which is why I have not merged this PR yet - we are investigating

**Comment by grkvlt** [comment]
@boydjohnson @chrisspanton @dcmiddle So, with the latest commits added to this PR the UI now works correctly, and @mikezaccardo and myself have got the server API integration tests mostly passing (in master they seem to be failing) so I will merge this shortly unless there are any objections.

**Pull request merged**